extensions/configuration/certificates.yaml

```diff
@@ -0,0 +1,134 @@
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: certificates.cert.gardener.cloud
+  namespace: kube-public
+  labels:
+    app.kubernetes.io/name: certificates.cert.gardener.cloud
+    busola.io/extension: resource
+    busola.io/extension-version: '0.5'
+data:
+  details: |
+    header:
+      - name: status
+        source: 'status.state ? status.state : "UNKNOWN"'
+        highlights:
+          informative:
+          - UNKNOWN
+        widget: Badge
+        description: status.message
+      - name: status.expirationDate
+        source: status.expirationDate
+      - name: commonName
+        source: spec.commonName
+    body:
+      - name: references
+        widget: Panel
+        children:
+          - name: issuer
+            source: status.issuerRef.name
+          - name: secret
+            source: spec.secretRef.name
+          - name: spec.csr
+            source: spec.csr
+          - name: metadata.annotations
+            source: metadata.annotations
+            widget: Labels
+          - name: spec.renew
+            source: spec.renew
+          - name: spec.dnsNames
+            source: spec.dnsNames
+            widget: Labels
+  form: |
+    - var: useCSR
+      dynamicValue: 'false'
+      simple: true
+      type: boolean
+      name: spec.useCSR
+    - simple: true
+      path: spec.commonName
+      required: true
+      placeholder: certificateCN
+      visibility: "$useCSR = false"
+    - simple: true
+      path: spec.csr
+      required: true
+      placeholder: spec.certSignReq
+      visibility: "$useCSR"
+      decodable: true
+      decodedPlaceholder: spec.certSignReqDecoded
+    - path: spec.dnsNames
+      widget: SimpleList
+      children:
+        - path: '[]'
+      placeholder: spec.additionalDomains
+    - name: issuer
+      path: spec.issuerRef
+      required: false
+      widget: ResourceRef      
+      resource:
+        kind: Secret
+        version: v1
+    - path: spec.renew
+    - var: useExistingSecret
+      name: spec.useExisting
+      dynamicValue: 'false'
+      type: boolean
+    - path: spec.secretName
+      visibility: "$useExistingSecret = false"
+    - path: spec.secretRef
+      widget: ResourceRef      
+      provideVar: secret
+      resource:
+        kind: Secret
+        version: v1
+      defaultExpanded: true
+      required: true 
```

Comment:
Secrets are fetched but they're filtered with a specific condition that is in the `SecretRef.js` file in busola.

```js
  const url = `/api/v1/secrets?labelSelector=${labelSelector ||
    ''}&fieldSelector=${fieldSelector || ''}`;
  const { data: secrets, loading } = useGetList()(url);
  ```
  
In `CertificateCreate.js` file, the component has some props:
```js
      <SecretRef
          advanced
          title={t('certificates.secret-ref')}
          tooltipContent={t('certificates.tooltips.secret-ref')}
          fieldSelector="type=kubernetes.io/tls"
          propertyPath="$.spec.secretRef"
          currentNamespace={namespace}
          required
        />
```  
On of them is fieldSelector which is the condition which Secrets are filtered by.

Reply:
Please, try to add similar condition.